cms/snapshot/CMS-DB.yaml

```diff
@@ -123,12 +132,13 @@ schema public:
     start_value: 1
   table collection:
     columns:
+    - displayName:
+        default: NULL::character varying
+        type: character varying(255)
```

Comment:
For some reason it messes up the order in my local environment, but I'm letting it slide because I don't know how to fix it and it's too much work every time to filter out the changes that are not intended.